src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -162,8 +162,42 @@ private void appendOneNativeArgument(ExecutionContext context, object obj, char
                     arg = PSObject.ToStringParser(context, ParserOps.Current(null, list));
                 }
 
+#if UNIX
+                Collection<string> resolvedPaths = new Collection<string>();
+#endif
                 if (!String.IsNullOrEmpty(arg))
                 {
+                    // This implements globbing of arguments given to native tools on UNIX.
+                    // Conversely, Windows tools implement their own globbing,
```

Comment:
@lzybkr wanted it tied to the platform in #1639. Do you want something in addition to `--%` to disable globbing?

I agree, inconsistent behavior isn't fun. But this is what I'm operating off:

> At a minimum, I think we can't do globbing on Windows at all - Windows tools implement globbing themselves, so even if we did something, it might work differently than the tool we're calling.

So Windows versus Linux tools are already introducing inconsistent behavior regardless of us.